src/AcmModal/AcmModal.test.tsx

```diff
@@ -3,36 +3,44 @@ import { render } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 import { axe } from 'jest-axe'
 import { AcmModal } from './AcmModal'
+import { AcmButton } from '../AcmButton/AcmButton'
+import { ButtonVariant } from '@patternfly/react-core'
 
 describe('AcmModal', () => {
-    test('renders in an open state', () => {
-        const onSubmit = jest.fn()
-        const onCancel = jest.fn()
-        const { getByRole, getByText } = render(
-            <AcmModal open={true} submit={onSubmit} cancel={onCancel} title="Modal title" message="Modal message" />
+    const onSubmit = jest.fn()
+    const onCancel = jest.fn()
+    const Component = () => {
+        return (
+            <AcmModal
+                isOpen={true}
+                title="Modal title"
+                actions={[
+                    <AcmButton key="confirm" variant={ButtonVariant.primary} onClick={onSubmit}>
+                        Submit
+                    </AcmButton>,
+                    <AcmButton key="cancel" variant={ButtonVariant.link} onClick={onCancel}>
+                        Cancel
+                    </AcmButton>,
+                ]}
+            >
+                Modal message
+            </AcmModal>
         )
+    }
+    test('renders in an open state', () => {
+        const { getByRole, getByText } = render(<Component />)
         userEvent.click(getByText('Submit'))
         userEvent.click(getByText('Cancel'))
         expect(getByRole('dialog')).toBeInTheDocument()
         expect(onSubmit).toHaveBeenCalled()
         expect(onCancel).toHaveBeenCalled()
     })
     test('does not render when in a closed state', () => {
-        const { container } = render(
-            <AcmModal
-                open={false}
-                submit={() => null}
-                cancel={() => null}
-                title="Modal title"
-                message="Modal message"
-            />
-        )
+        const { container } = render(<Component />)
```

Comment:
I'm confused as to how this is closed when `Component` has `isOpen={true}`.

Reply:
Good point; why are these tests passing 😂